### DISCUSSION TITLE
Sample rates other than 8000 Hz and 16000 Hz appear to work

### DISCUSSION THREAD

**Discussion by Jellun** [snakers4/silero-vad#377] [Q&A] [answered]
Hi all,

The README states that Silero VAD supports 8000 Hz and 16000 Hz sampling rates. However, in the Colaboratory notebook provided, I can upload wav files in 48000 sample rate and I can change SAMPLING_RATE = 48000. It all appear to work. Can anyone explain why?

Kind thanks,
Jun

**Comment by snakers4** [comment, marked as answer]
For such sample rates, each nth sample is taken.

**Comment by Jellun** [comment]
Do you mean that Silero-vad internally converts higher sampling rates down to 16000Hz before it performs vad?

**Reply by snakers4** [reply]
For sample rates proportional to 16k it selects each n-th sample.
